app/assets/stylesheets/welcome.scss

```diff
@@ -0,0 +1,51 @@
+$gold: #F8A803;
+
+.welcome-wrapper {
+  background-image: image-url('landing-page-pic.jpg');
+  background-position: center;
+  background-size: cover;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, background-size, height